Task1/readFile.js

```diff
@@ -0,0 +1,54 @@
+// READ FILE RECURSIELY & Create Hash. Also store result to Output file
+
+const fs = require('fs');
+const path = require('path');
+const crypto = require('crypto');
+const outputFile = "result.txt";
+const myArgs = process.argv.slice(2);
+const myPath = myArgs[0];
```

Comment:
use relevant variable names. instead of myPath say directoryPath.

also as an improvement you could accept output file as second argument.